docs-md/interfaces/GetOwnersForNftResponse.md

```diff
@@ -18,6 +18,12 @@ The response object for the [getOwnersForNft](../classes/NftNamespace.md#getowne
 
 An array of owner addresses for the provided token.
 
+### pageKey (optional)
```

Comment:
Can you revert this change? We usually only regenerate the SDK documentation when creating a new release